client/.eslintrc.js

```diff
@@ -1,29 +1,32 @@
 module.exports = {
-  root: true,
-  env: {
-    node: true
-  },
-  extends: [
-    'plugin:vue/essential',
-    '@vue/typescript/recommended',
-    '@vue/prettier/@typescript-eslint'
-  ],
```

Comment:
Did this file get changed on purpose? I think these extensions are the old ones, before I removed them. They were culprits that messed up our prettier settings.

Reply:
Oh ok, I think I must've messed up the merge. I didn't change that